<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>660</width>
    <height>552</height>
   </rect>
  </property>
  <property name="acceptDrops">
   <bool>false</bool>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <layout class="QGridLayout" name="gridMainLayout">
    <property name="bottomMargin">
     <number>9</number>
    </property>
    <item row="1" column="0">
     <widget class="Line" name="lineDivider">
      <property name="orientation">
       <enum>Qt::Horizontal</enum>
      </property>
     </widget>
    </item>
    <item row="2" column="0">
     <widget class="QTableWidget" name="tblFiles">
      <property name="acceptDrops">
       <bool>true</bool>
      </property>
      <column>
       <property name="text">
        <string>Source</string>
       </property>
      </column>
      <column>
       <property name="text">
        <string>Name</string>
       </property>
      </column>
      <column>
       <property name="text">
        <string>Type</string>
       </property>
      </column>
      <column>
       <property name="text">
        <string>Size</string>
       </property>
      </column>
      <column>
       <property name="text">
        <string>Destination</string>
       </property>
      </column>
     </widget>
    </item>
    <item row="0" column="0">
     <widget class="QFrame" name="frmContainer">
      <property name="frameShape">
       <enum>QFrame::StyledPanel</enum>
      </property>
      <property name="frameShadow">
       <enum>QFrame::Plain</enum>
      </property>
      <layout class="QHBoxLayout" name="horizontalLayout">
       <property name="margin">
        <number>0</number>
       </property>
       <item>
        <widget class="QGroupBox" name="grpDestBox">
         <property name="title">
          <string>Destination</string>
         </property>
         <property name="flat">
          <bool>true</bool>
         </property>
         <property name="checkable">
          <bool>false</bool>
         </property>
         <layout class="QGridLayout" name="gridDestBoxLayout">
          <item row="0" column="1">
           <widget class="QLineEdit" name="txtCopyToDir">
            <property name="dragEnabled">
             <bool>true</bool>
            </property>
            <property name="placeholderText">
             <string>Copy to directory</string>
            </property>
           </widget>
          </item>
          <item row="0" column="2">
           <widget class="QPushButton" name="btnCopyToDir">
            <property name="cursor">
             <cursorShape>PointingHandCursor</cursorShape>
            </property>
            <property name="text">
             <string>...</string>
            </property>
           </widget>
          </item>
          <item row="0" column="0">
           <widget class="QLabel" name="lblCopyTo">
            <property name="text">
             <string>Copy To</string>
            </property>
           </widget>
          </item>
          <item row="1" column="1">
           <widget class="QSpinBox" name="bxDirLevel"/>
          </item>
          <item row="1" column="0">
           <widget class="QLabel" name="lblDirLevel">
            <property name="text">
             <string>Preserv Level</string>
            </property>
           </widget>
          </item>
          <item row="2" column="1" colspan="2">
           <widget class="QProgressBar" name="targetStatus">
            <property name="value">
             <number>0</number>
            </property>
            <property name="alignment">
             <set>Qt::AlignLeading|Qt::AlignLeft|Qt::AlignVCenter</set>
            </property>
           </widget>
          </item>
         </layout>
        </widget>
       </item>
       <item>
        <widget class="Line" name="line">
         <property name="orientation">
          <enum>Qt::Vertical</enum>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QFrame" name="frmPlayer">
         <property name="styleSheet">
          <string notr="true"/>
         </property>
         <property name="frameShape">
          <enum>QFrame::StyledPanel</enum>
         </property>
         <property name="frameShadow">
          <enum>QFrame::Plain</enum>
         </property>
         <layout class="QVBoxLayout" name="vertPlayerLayout">
          <property name="margin">
           <number>0</number>
          </property>
          <item>
           <widget class="QDial" name="dialVolume"/>
          </item>
          <item>
           <widget class="QFrame" name="frmSeeker">
            <property name="frameShape">
             <enum>QFrame::StyledPanel</enum>
            </property>
            <property name="frameShadow">
             <enum>QFrame::Raised</enum>
            </property>
            <layout class="QHBoxLayout" name="horzSeekerLayout">
             <property name="leftMargin">
              <number>0</number>
             </property>
             <property name="rightMargin">
              <number>3</number>
             </property>
             <item>
              <widget class="QPushButton" name="btnPlay">
               <property name="text">
                <string>Play</string>
               </property>
               <property name="iconSize">
                <size>
                 <width>24</width>
                 <height>24</height>
                </size>
               </property>
               <property name="autoDefault">
                <bool>false</bool>
               </property>
               <property name="default">
                <bool>false</bool>
               </property>
               <property name="flat">
                <bool>false</bool>
               </property>
              </widget>
             </item>
             <item>
              <widget class="Phonon::SeekSlider" name="seekPlayer"/>
             </item>
            </layout>
           </widget>
          </item>
         </layout>
        </widget>
       </item>
      </layout>
     </widget>
    </item>
    <item row="3" column="0">
     <widget class="QFrame" name="frmCommand">
      <property name="frameShape">
       <enum>QFrame::StyledPanel</enum>
      </property>
      <property name="frameShadow">
       <enum>QFrame::Plain</enum>
      </property>
      <layout class="QHBoxLayout" name="horzCommandLayout">
       <property name="leftMargin">
        <number>0</number>
       </property>
       <property name="topMargin">
        <number>9</number>
       </property>
       <property name="rightMargin">
        <number>0</number>
       </property>
       <property name="bottomMargin">
        <number>0</number>
       </property>
       <item>
        <widget class="QPushButton" name="btnUndoCopy">
         <property name="enabled">
          <bool>false</bool>
         </property>
         <property name="text">
          <string>Undo</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QPushButton" name="btnPauseCopy">
         <property name="text">
          <string>Pause Copy</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QPushButton" name="btnStartCopy">
         <property name="text">
          <string>Start Copy</string>
         </property>
        </widget>
       </item>
      </layout>
     </widget>
    </item>
   </layout>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>660</width>
     <height>27</height>
    </rect>
   </property>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
 </widget>
 <customwidgets>
  <customwidget>
   <class>Phonon::SeekSlider</class>
   <extends>QWidget</extends>
   <header location="global">phonon/seekslider.h</header>
  </customwidget>
 </customwidgets>
 <resources/>
 <connections/>
</ui>
